src/event.cairo

```diff
@@ -84,40 +102,58 @@ mod StarkPassEvent {
             self.ticket_price.read()
         }
 
-        // ##### ERC20 ACTIVE FUNCTIONS #####
+        fn change_organizer(ref self: ContractState) {
+            // Only the current organizer can assign a new organizer
+            self.only_owner();
+
+            // Change the owner
+            let new_owner = get_caller_address();
```

Comment:
Shouldn't the new owner be somebody else? If we assign the caller address it wouldn't change the organizer since only organizer is able to call this function

Reply:
You're completely right. Lack of sleep still kicking in haha. Ty for catching